src/platform/baytrail/platform.c

```diff
@@ -65,16 +66,20 @@ static const struct sof_ipc_fw_ready ready
 	.flags = DEBUG_SET_FW_READY_FLAGS
 };
 
-#define NUM_BYT_WINDOWS		6
-static const struct sof_ipc_window sram_window
-	__section(".fw_ready_metadata") = {
+EXT_MAN_PORT(
+	EXT_MAN_ELEM_WINDOW,
+	sizeof(struct ext_man_windows),
+	window,
+	static const struct ext_man_windows xsram_window,
+	static const struct sof_ipc_window sram_window
+		__section(".fw_ready_metadata"),
+	_META_EXPAND(
 	.ext_hdr	= {
 		.hdr.cmd = SOF_IPC_FW_READY,
-		.hdr.size = sizeof(struct sof_ipc_window) +
-			sizeof(struct sof_ipc_window_elem) * NUM_BYT_WINDOWS,
+		.hdr.size = sizeof(struct sof_ipc_window),
 		.type	= SOF_IPC_EXT_WINDOW,
 	},
-	.num_windows	= NUM_BYT_WINDOWS,
+	.num_windows	= 6,
```

Comment:
Why would we do this ?

Reply:
After change, this macro was used only once - in this place

Reply:
Doesn't matter how many times it used, it's a needless and unrelated change here.

Reply:
updated